ClientApp/src/NavBar/NavBar.scss

```diff
@@ -0,0 +1,100 @@
+@import '../Styles/constants';
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+li {
+  float: left;
+}
+
+.page-headers{
+  margin-left: 12.5%;
+}
+  
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+  
+li a:hover:not(.active) {
+  background-color: #111;
+}
+  
+.active {
+  background-color: $color-primary;
+}
+
+.mobile-nav-bar-container{
+  margin-bottom: 70px;
+}
+
+.mobile-nav-bar {
+  overflow: hidden;
+  background-color: #333;
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+.mobile-nav-bar-title {
+  color: white;
+
+}
+
+.mobile-nav-bar #myLinks {
+  display: none;
+}
+
+.mobile-nav-bar a {
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: block;
+}
+
+.icon {
+  display: block;
+  position: fixed;
+  top: 0;
+  right: 0;
+  color: white;
+}
+.closed-nav-bar{
+  display: none;
+}
+
+.mobile-nav-bar a:hover {
+  background-color: $color-primary;
+  color: black;
+}
+
+.active {
+  background-color: $color-primary;
+  color: white;
+}
+
+@media only screen and (min-width: $two-column-max-pixels){
```

Comment:
the two-column-max-pixels style is up at 1000px which is actually bigger than all tablets and the size of some smaller laptop screens, this might want to be down at the tablet breakpoint which is more like 768 just for consistency between devices - do feel free to disagree if 1000px is something everyone has chosen to work for styling

Reply:
We discussed this as group and landed on 1000px because it was nice number. Our idea was that the two column display and horizontal navbar would be used for iPad pro and desktop. Anything smaller would use the mobile navbar :)

Reply:
sounds good, I'll resolve my comment :)